docs/src/DeveloperDocumentation/way_of_working.md

```diff
@@ -0,0 +1,135 @@
+# Way of Working
+This document contains the essentials to get you started coding. Please refer
+to the [Introduction for new developers](@ref) if you are new to OSCAR.
+
+!!! note "Pay attention to your GitHub notifications!"
+    Once you open a pull request on GitHub you will receive feedback, comments,
+    and questions on GitHub. So please pay attention to your GitHub
+    notifications.
+
+## Important notes
+1. If you encounter error messages after rebasing to the current master, chances
+   are that some dependencies need upgrading. Please first try whether
+   executing `]up` gets rid of your errors.
+2. Please have a look at the [Developer Style Guide](@ref). Adhering to the
+   style guide makes reviewing code easier for us, and hence your new feature
+   can be merged faster.
+3. Let us know what you are working on:
+   - You can open a draft pull request on GitHub right at the beginning of your
+     work.
+   - Feel free to contact us on
+     [Slack](https://join.slack.com/t/oscar-system/shared_invite/zt-thtcv97k-2678bKQ~RpR~5gZszDcISw).
+   - Have a look at [our community page](https://oscar.computeralgebra.de/community/).
+4. Please also read our page on [Documenting OSCAR code](@ref).
+5. Look at existing code that does similar things to your project to get an
+   idea of what OSCAR code should look like. Try to look at multiple examples.
+
+## Overview
+In general you have to do
+the following six steps for submitting changes to the Oscar source:
+
+1. Fork the main Oscar.jl repository. For this go to the [Oscar.jl GitHub
+   page](https://github.com/oscar-system/Oscar.jl) and click on "Fork" at the
+   upper right.
+2. Clone your forked repository to your local machine.
+   ```
+   git clone git@github.com:your_github_username/Oscar.jl
```

Comment:
Do we expect / require everyone to work via SSH? Perhaps we should instead / alternatively link to the official GitHub docs for forking and cloning and all that?

Reply:
I added some links, but I wanted to provide something minimal for copy-pasting.